### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewDebug_ScreenReader: JAWS does not read the state of the toolbars as "expanded/collapsed" when focus moves to them using "Tab" key.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52308]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code while JAWS is ON.
2. Navigate to Activity Bar and select "View Debug "(Cntrl+Shift+D) button.
3. Navigate to toolbars section under heading "Debug".

**Actual:**
JAWS does not read the state of the toolbars as "expanded/collapsed" when focus moves to them using "Tab" key. It only reads the states when any action is provided on them.

**Expected:**
JAWS should read state of the combobox as "expanded/collapsed" when focus moves to them using "Tab" key.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the state of the combobox when focus moves to it. They need to perform action to know what state it was previously in.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_ViewDebug_JAWS_StateIssue.pptx](https://github.com/Microsoft/vscode/files/2114763/A11y_VSCode_ViewDebug_JAWS_StateIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
